### PR TITLE
[JW8-11686] Fix typo

### PR DESCRIPTION
**Pull request by mamaddox**
### This PR will...
- Fix `Browser.Safari` case typo, should be lower case
### Why is this Pull Request needed?
bugfix
### Are there any Pull Requests open in other repos which need to be merged with this?
Relates to: https://github.com/jwplayer/jwplayer/pull/3845
#### Addresses Issue(s):

JW8-11686

### Checklist
- [ ] Jenkins builds and unit tests are passing
- [ ] I have reviewed the automated results

### CODE REVIEW

**Comment by jwrobot-player** [comment]
[:warning:](https://jenkins-player.longtailvideo.com/job/jw7-pr-multi-opensource/1788/) MULTI Build for commit db29132012b5abc809bc442be0130ed56d1fbb07 is unstable (UNSTABLE).
[:building_construction:](https://jenkins-player.longtailvideo.com/job/jw7-opensource/9182/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-opensource/9182/consoleFull) jwplayer build SUCCESS
[:clipboard:](https://jenkins-player.longtailvideo.com/job/jw7-opensource-unit-tests/6155/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-opensource-unit-tests/6155/consoleFull) jwplayer unit tests SUCCESS
[:building_construction:](https://jenkins-player.longtailvideo.com/job/jw7-commercial/28043/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-commercial/28043/consoleFull) jwplayer-commercial build SUCCESS
[:clipboard:](https://jenkins-player.longtailvideo.com/job/jw7-commercial/23539/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-commercial-unit-tests/23539/consoleFull) jwplayer-commercial unit tests SUCCESS
[:cucumber: Cucumber Integration Tests](https://jenkins-player.longtailvideo.com/job/jw7-commercial-cucumber-wdio/5763/allure/) UNSTABLE
[:jack_o_lantern:](http://player-pr-test-jenkins.longtailvideo.com/builds/28043/archive/test/squash/) Manual Tests
[:tv:](http://player-pr-test-jenkins.longtailvideo.com/builds/28043/archive/test/view/) Views

**Review by nebnitt — APPROVED** [review]

File: src/js/providers/html5.ts [hunk 1]
```diff
@@ -101,7 +101,7 @@ function VideoProvider(this: HTML5Provider, _playerId: string, _playerConfig: Ge
     let { loadAndParseHlsMetadata, minDvrWindow } = _playerConfig;
 
     // Toggle for a bug in iOS and Safari where you are unable to seek in initially muted streams
-    const iosMuteToggle = OS.iOS || Browser.Safari;
+    const iosMuteToggle = OS.iOS || Browser.safari;
```
Comment (robwalch) [inline src/js/providers/html5.ts:104]: Are we not running type checks on build? This should have been caught by TypeScript. The build should have failed. Both html5 and environment are .ts files.

**Review by robwalch — CHANGES_REQUESTED** [review]
This is a good example of untyped TypeScript. Please the exports from environment.ts.

It's half done:
```
/**
 * @typedef {object} BrowserEnvironment
 * @property {boolean} androidNative - Is the browser Android Native?
 * @property {boolean} chrome - Is the browser Chrome?
 * @property {boolean} edge - Is the browser Edge?
 * @property {boolean} facebook - Is the browser a Facebook webview?
 * @property {boolean} firefox - Is the browser Firefox?
 * @property {boolean} ie - Is the browser Internet Explorer?
 * @property {boolean} msie - Is the browser MSIE?
 * @property {boolean} safari - Is the browser Safari?
 * @property {EnvironmentVersion} version - The browser version.
 */
export const Browser: GenericObject = {};
```

but this would be better written as (no interface definitions required):

```
export const Browser = {
    get androidNative(): boolean {
        return isAndroidNative();
    }
    // ...
};
```
We really don't need the memoization, and declaring as a static object defines the types in place and sets everything up on script parse without additional runtime cost.

**Comment by mamaddox** [comment]
@robwalch I put the changes in a separate PR since they were larger than I thought and to isolate this bugfix so it wouldn't be dependent on them. https://github.com/jwplayer/jwplayer/pull/3847

**Review by nebnitt — APPROVED** [review]
Typing addressed in separate PR - let's get this merged so the fix is in and follow-up with the typing work.

**Comment by mamaddox** [comment]
Closing because fix is part of: https://github.com/jwplayer/jwplayer/pull/3847

**Pull request closed without merging**